geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/status/GetTypeNamesCommand.scala

```diff
@@ -8,15 +8,29 @@
 
 package org.locationtech.geomesa.tools.status
 
+import com.beust.jcommander.ParameterException
 import org.geotools.data.DataStore
 import org.locationtech.geomesa.tools.{Command, DataStoreCommand}
 
+import scala.util.control.NonFatal
+
 trait GetTypeNamesCommand[DS <: DataStore] extends DataStoreCommand[DS] {
 
   override val name = "get-type-names"
 
   override def execute(): Unit = {
-    Command.output.info("Current features types:")
-    withDataStore(_.getTypeNames.foreach(Command.output.info))
+    getTypeNames
+  }
+
+  @throws[ParameterException]
+  def getTypeNames = {
```

Comment:
maybe instead of changing this here, we could make the `withDataStore` method check that the data store isn't null and throw a ParameterException if it is. That way we'd be covered in all the commands. Would handle the VersionRemoteCommand too.